Makefile

```diff
@@ -2,6 +2,8 @@
 # Image URL to use all building/pushing image targets
 IMG ?= numaplane-controller
 VERSION ?= latest
+# Default cluster name where pumaplane get deployed, update it as needed.
```

Comment:
fix spelling of "numaplane"

Reply:
somehow spell check is not working in my editor, I need to fix it.